### DISCUSSION TITLE
Training datasheet Yolov3 Problem (Only detection for person)

### DISCUSSION THREAD

**Discussion by fais1901** [community/community#158245] [Projects and Issues]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I have a problem in my project (detection only person), when training the model with the datasheet the readings are iou_loss=nan, total_loss=nan and some even show -nan(ind) and yes one more rewritten_bbox=0.000% (throughout training). If anyone knows please help me, thanks 

And here I attach a screenshot of the training 
![training yolov3](https://github.com/user-attachments/assets/241c7eaa-4204-4f2f-b8b5-add8706d5b64)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by jonhubby** [comment]
Usually means something's going wrong early in training, could be due to issues like incorrect annotations, image sizes not matching what the model expects, or a learning rate that's too high. You might want to double-check your dataset format (especially bounding box values) and try lowering the learning rate to see if that stabilizes the loss.

**Reply by fais1901** [reply]
Actually, I took this datasheet from the datasheet provider's website and I have also checked the order and it is correct, namely object class, x, y, w, h, so the annotation shouldn't be a problem, but can you explain about reducing the learning rate?

**Comment by github-actions** [comment]
🕒 **Discussion Activity Reminder** 🕒

This Discussion has been labeled as dormant by an automated system for having no activity in the last 60 days. Please consider one the following actions:

1️⃣ Close as Out of Date: If the topic is no longer relevant, close the Discussion as `out of date` at the bottom of the page.

2️⃣ Provide More Information: Share additional details or context — or let the community know if you've found a solution on your own.

3️⃣ Mark a Reply as Answer: If your question has been answered by a reply, mark the most helpful reply as the solution.

Note: This dormant notification will only apply to Discussions with the `Question` label. To learn more, see our [recent announcement](https://github.com/orgs/community/discussions/70478).

Thank you for helping bring this Discussion to a resolution! 💬
